cetmix_tower_server/models/cx_tower_variable_option.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2022 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import fields, models
+
+
+class TowerVariableOption(models.Model):
+    _name = "cx.tower.variable.option"
+    _description = "Cetmix Tower Variable Options"
+
+    name = fields.Char(string="Option Value", required=True)
+    variable_id = fields.Many2one(
+        comodel_name="cx.tower.variable",
+        string="Variable",
+        required=True,
+        ondelete="cascade",
+        default=lambda self: self.env.context.get("default_variable_id"),
```

Comment:
Question, not an issue: why do we need to take the default value from context?

Reply:
Because I was using in a view and the field was hidden and to be sure that the field would get that variable_id. I will fix it because I dont need to have it.